esp/src/src-react/components/Menu.tsx

```diff
@@ -294,9 +294,10 @@ export const SubNavigation: React.FunctionComponent<SubNavigationProps> = ({
             <Stack.Item align="center" grow={1}>
                 <Stack horizontal>
                     <Stack.Item grow={0} className={navStyles.wrapper}>
-                        {subMenuItems[mainNav]?.map(row => {
+                        {subMenuItems[mainNav]?.map((row, idx) => {
                             return <Link
                                 disabled={row.itemKey === "/topology/logs" && logsDisabled}
+                                key={`MenuLink_${idx}`}
```

Comment:
This isn't really related, but I just noticed the React error in the console about the child components not having unique keys.